flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/RemoteInputChannel.java

```diff
@@ -216,7 +220,7 @@ void retriggerSubpartitionRequest(int subpartitionIndex) throws IOException {
 	}
 
 	@Override
-	public List<Buffer> requestInflightBuffers(long checkpointId) throws IOException {
+	public void requestInflightBuffers(long checkpointId, ChannelStateWriter channelStateWriter) throws IOException {
```

Comment:
rename all of the methods `requestInflightBuffers` -> `spillInflightBuffers`? As this method is no longer returning/requesting but spilling inflight buffers?

Reply:
Thank for the review. I have renamed the method ```requestInflightBuffers``` to ```spillInflightBuffers```.